src/ThermalScattering.tex

```diff
@@ -125,7 +125,32 @@ \subsubsection{\textsf{ITXE} Block}\label{sec:ITXEBlock}
 
 Because the use of discrete outgoing energies and cosines can result in unphysical spikes in the neutron flux spectrum at thermal energies, some Monte Carlo codes attempt to ``smear'' the outgoing energies and cosines to produce a smoother distribution (that more closely approximates a continuous distribution with $\nxs{7}=2$).
 
-\todo[inline]{Describe the format for $\nxs{7}=2$.}
+When $\nxs{7}=2$, the distribution of outgoing energies for each incident energy is continuous in energy and specified by a probability density function and cumulative distribution function. The format of the \Block{ITXE} in this case is given in \Tableref{tab:ITXEBlockContinuousHeader} and \Tableref{tab:ITXEBlockContinuousData}. As before, the index at the start of the block is \startblock{ITXE}=\jxs{3}. Unlike in the $\nxs{7} < 2$ cases, the number of outgoing energies for each incident energy is allowed to vary. The number of discrete cosines, $N_\mu=$\nxs{3}-1, remains the same for each pair of incident and outgoing energies, however.
+
+\begin{XSSTable}{Start of \Block{ITXE} for $\nxs{7}=2$}
+  \startblock{ITXE}          & \var{L}$(l),l=1,\ldots,N_{in}$  & Location in \XSS\ of distribution for incident energy $l$ \\
```

Comment:
It might be good to remind (perhaps a table footnote?) people what `N_{in}` is. I had to go looking for it because I wasn't sure.

Reply:
Good point, that had also occurred to me